modules/avro-exporter/src/main/scala/io/janstenpickle/trace4cats/avro/AvroSpanExporter.scala

```diff
@@ -151,27 +152,30 @@ object AvroSpanExporter {
         .drain
 
     for {
-      avroSchema <- Resource.liftF(AvroInstances.completedSpanSchema[F])
-      address <- Resource.liftF(Sync[F].delay(new InetSocketAddress(host, port)))
-      queue <- Resource.liftF(InspectableQueue.bounded[F, Batch[G]](1))
-      semaphore <- Resource.liftF(Semaphore[F](Long.MaxValue))
-      socketGroup <- TCPSocketGroup(blocker)
+      avroSchema <- Resource.eval(AvroInstances.completedSpanSchema[F])
+      h <- Resource.eval(Host.fromString(host).liftTo[F](new IllegalArgumentException(s"invalid host $host")))
+      p <- Resource.eval(Port.fromInt(port).liftTo[F](new IllegalArgumentException(s"invalid port $port")))
+      address = SocketAddress(h, p)
+      queue <- Resource.eval(Queue.bounded[F, Batch[G]](1))
+      semaphore <- Resource.eval(Semaphore[F](Long.MaxValue))
+      socketGroup <- Network[F].socketGroup()
```

Comment:
`Blocker` is not needed anymore. But threads under `socketGroup` and `datagramSocketGroup` can be configured.

Reply:
No configuration needed. Default instances are ok.